RedefineGuide/Controllers/SurgeryController.swift

```diff
@@ -103,24 +128,27 @@ class SurgeryController: NSObject {
             return
         }
         
-        // Use the camera's transform to get its current orientation and position
-        let transform = currentFrame.camera.transform
-        let cameraPosition = SCNVector3(transform.columns.3.x, transform.columns.3.y, transform.columns.3.z)
-        
-        // Calculate the forward vector from the camera transform
-        let forward = SCNVector3(-transform.columns.2.x, -transform.columns.2.y, -transform.columns.2.z)
-        let adjustedForward = forward.normalized() * 0.5 // Adjust to be 0.5 meters in front
+//        // *** Fixed cad model relative to the camera
+//        let cameraTransform = currentFrame.camera.transform
+//        let cameraPosition = SCNVector3(cameraTransform.columns.3.x, cameraTransform.columns.3.y, cameraTransform.columns.3.z)
+//        let forward = SCNVector3(-cameraTransform.columns.2.x, -cameraTransform.columns.2.y, -cameraTransform.columns.2.z)
+//        let adjustedForward = forward.normalized() * 0.9 // Adjust to be 0.5 meters in front
+//        overlayModel.position =  adjustedForward + cameraPosition
+//
+//        let cameraQuaternion = simd_quaternion(cameraTransform)
+//        let scnQuaternion = SCNQuaternion(cameraQuaternion.vector.x, cameraQuaternion.vector.y, cameraQuaternion.vector.z, cameraQuaternion.vector.w)
+//        overlayModel.orientation = scnQuaternion
+//        // *** Fixed cad model relative to the camera
         
-        overlayModel.position =  adjustedForward + cameraPosition
+        // *** Fixed CAD model relative to world
```

Comment:
Isn't the whole point of this function to keep the CAD model in front of the phone so the surgeon can line up the CAD model with the actual bone?  Wouldn't this break that?